tests/unit/test_app.py

```diff
@@ -0,0 +1,2 @@
+def test_sum():
```

Comment:
[question] why this test exists?

Reply:
Just to help us configure the pipeline correctly, will be removed in the future